server/bin/liquibase.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Source functions library
+_prefer_jre="true"
+. /usr/share/java-utils/java-functions
+
+# Source system prefs
+if [ -f /etc/java/liquibase.conf ] ; then
+  . /etc/java/liquibase.conf
+fi
+
+# Source user prefs
+if [ -f $HOME/.liquibaserc ] ; then
+  . $HOME/.liquibaserc
+fi
+
+# Configuration
+MAIN_CLASS=liquibase.integration.commandline.Main
+BASE_FLAGS=""
+BASE_OPTIONS=""
+
+CP_CLASSPATH=${CP_LIQUIBASE_CLASSPATH:-/var/lib/tomcat/webapps/candlepin/WEB-INF/lib}
+CLASSPATH=$(JARS=("$CP_CLASSPATH"/*.jar); IFS=:; echo "${JARS[*]}")
+
+# Set parameters
+set_jvm
+set_classpath "${CLASSPATH}"
```

Comment:
Please remove the `set_classpath` line. Apparently just setting `CLASSPATH` environment variable, as you did on line 23 is sufficient. Removing this line also fixes the

```
/usr/bin/build-classpath: Unable to find xmvn-resolve.
/usr/bin/build-classpath: Make sure that xmvn-resolve package is installed.
/usr/bin/build-classpath: error: Some specified jars were not found
```

messages.

Reply:
Done.